apps/demos/Demos/Chat/Customization/jQuery/styles.css

```diff
@@ -0,0 +1,53 @@
+.dx-viewport {
+    padding: 20px;
+    min-width: 780px;
+    border: 1px solid var(--dx-color-border);
+    border-radius: 8px;
+  }
```

Comment:
Kindly fix indents, add empty line at the end of the file

Reply:
[a788963](https://github.com/DevExpress/DevExtreme/pull/28441/commits/a7889633ac3296ccaa25a0589dccff72c053b179)